apps/src/feedback.js

```diff
@@ -328,6 +348,27 @@ FeedbackUtils.prototype.getNumCountableBlocks = function() {
   return this.getCountableBlocks_().length;
 };
 
+FeedbackUtils.prototype.buildPuzzleRatingButtons_ = function () {
+  var buttonContainer = document.createElement('div');
+  buttonContainer.id = 'puzzleRatingButtons';
+  buttonContainer.innerHTML = require('./templates/puzzleRating.html.ejs')();
+
+  var buttons = buttonContainer.querySelectorAll('.puzzle-rating-btn');
+  var buttonClickHandler = function () {
+    for (var _ = 0, button; (button = buttons[_]); _++) {
+      if (button != this) {
+        button.classList.remove('enabled');
```

Comment:
dangit, and here I thought I was being all clever using the native funtions (╯°□°）╯︵ 6ƎI